app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  # belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def self.update_likes_counter(post_id)
+    post = Post.find(post_id)
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] As the previos reviewer mentioned kindly make sure to change the `update_like_counter` method just like what you did in the `comment` model
```ruby
def update_likes_counter(post_id)
   post.increment!(:likes_counter)
end
```